ArcGISRuntimeSDKQt_QMLSamples/Maps/ManageBookmarks/ManageBookmarks.qml

```diff
@@ -56,6 +58,41 @@ Rectangle {
                 map.bookmarks.append(bookmark);
             }
         }
+
+        // Create the add button so new bookmarks can be added
+        Rectangle {
+            id: addButton
+            property bool pressed: false
+            anchors {
+                right: parent.right
+                bottom: mapView.attributionTop
+                rightMargin: 20
+                bottomMargin: 20
+            }
+            width: childrenRect.width
+            height: childrenRect.height
+            color: pressed ? "#959595" : "#D6D6D6"
+            radius: 100
+            border {
+                color: "#585858"
+                width: 1
+            }
+
+            Image {
+                rotation: 45
+                source: "qrc:/Samples/Maps/ManageBookmarks/add.png"
```

Comment:
i don't think this will render correctly on all platforms if you don't define sizes. it'll render raw size which varies a lot across devices